src/libpgmoneta/message.c

```diff
@@ -912,6 +912,36 @@ pgmoneta_create_replication_slot_message(char* create_slot_name, struct message*
    return MESSAGE_STATUS_OK;
 }
 
+int
+pgmoneta_create_search_replication_slot_message(char* slot_name, struct message** msg)
+{
+   char cmd[1024];
+   struct message* m = NULL;
+   size_t size;
+
+   memset(&cmd[0], 0, sizeof(cmd));
+
+   snprintf(cmd, sizeof(cmd), "SELECT slot_name, slot_type FROM pg_replication_slots WHERE slot_name = '%s';", slot_name);
```

Comment:
We don't need the `slot_type` since we already know it will be a `physical` one

Reply:
What if the user configured the wrong slot type?

Reply:
Sure, we can add that check - ideally users should use `create_slot = yes` to control slots